static/src/js/util/request/admin/metricsPreferencesRequest.js

```diff
@@ -0,0 +1,15 @@
+import Request from '../request'
+import { getEnvironmentConfig } from '../../../../../../sharedUtils/config'
+
+export default class MetricsPreferencesRequest extends Request {
```

Comment:
Everywhere else in the code this is referred to has Preferences Metrics, but this class is Metrics Preferences. Keep the naming consistent and switch this to Preferences before Metrics

Reply:
@macrouch The front end is referred to as Preferences Metrics, but we have another metrics stuff called Metrics Retrievals so I was going off that. Would you like me to change that to match up as well? The reason it's nice to have metrics first is that the spelling/alphabetical order you can see all the metrics together as apposed to them being set apart from one another when they do similar functionality.

Reply:
I'm just looking for consistency throughout the code. The component that uses it is "AdminPreferencesMetrics", the actions are "preferencesMetrics", the lambda is "adminGetPreferencesMetrics", but this request class is "metricsPreferencesRequest". 

If I wanted to find all the code related to this subject (for debugging/tracing/refactoring/etc.), I would do a project wide search for "preferencesMetrics", but that wouldn't return these files (making the above tasks more difficult).